assets/app/scripts/directives/quotaUsageChart.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+
+angular.module('openshiftConsole')
+  .directive('quotaUsageChart', function($filter, Logger) {
+    return {
+      restrict: 'E',
+      scope: {
+        used: '=',
+        total: '=',
+        // 'cpu' or 'memory'
+        type: '@',
+        // Defaults to 'bottom'.
+        // http://c3js.org/reference.html#legend-position
+        legendPosition: '@?'
+      },
+      // Replace the element so it can be centered using class="center-block".
+      replace: true,
+      templateUrl: 'views/_quota-usage-chart.html',
+      link: function($scope, element) {
+        var usageValue = $filter('usageValue');
+        var usageWithUnits = $filter('usageWithUnits');
+        function updateCenterText() {
+          var donutChartTitle = d3.select(element[0]).select('text.c3-chart-arcs-title');
+          if (!donutChartTitle) {
+            Logger.warn("Can't select donut title element");
+            return;
+          }
+
+          var valueAndType = usageWithUnits($scope.total, $scope.type).split(' ');
```

Comment:
do we use this anywhere else like this? worried that usageWithUnits is a display method, if we ever decided to strip the space out then this would break